examples/capable/src/bpf/capable.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+
+#ifndef CAPABLE_CAPABLE_H
+#define CAPABLE_CAPABLE_H
+
+#define TASK_COMM_LEN 16
+#define BPF_MAX_STACK_DEPTH 127
+
+struct event {
+    gid_t tgid;
+    pid_t pid;
+    uid_t uid;
+    int cap;
+    int audit;
+    int insetid;
+    u8 comm[TASK_COMM_LEN];
+    int kernel_stack_id;
+    int user_stack_id;
+};
+
+enum Unique {UnqOff, UnqPid, UnqCgroup};
```

Comment:
Nit: kernel naming style
```suggestion
enum uniqueness {UNQ_OFF, UNQ_PID, UNQ_CGROUP};
```

Reply:
@danobi I had it like this first, but when I ``cargo build``, I got warnings for [enum naming conventions](https://doc.rust-lang.org/rust-by-example/custom_types/enum.html). When I thought this was to be a rust project, I thought it might be easier to have it coded rust style in the c code as well so that no compile time warnings happened. I wasn't sure if code-gen should be changed to emit rust convention code or do [this](https://stackoverflow.com/a/64363119/2277013) instead. I'm totally fine changing it back and ignoring that compile time warning. Lmk.

Reply:
Oh, maybe we can add more stuff to https://github.com/libbpf/libbpf-rs/blob/3ff6c9377e4d98c4af700347a99e303e9abff319/libbpf-cargo/src/gen.rs#L666-L670 . Having C code look like C ought to take precedence here. B/c the rust code knows it's using C types but we cannot say the same about the other way around.